WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -245,7 +248,8 @@ public void SelectWallet(string walletName)
 		[JsonRpcMethod("stop")]
 		public async Task StopAsync()
 		{
-			await Global.DisposeAsync().ConfigureAwait(false);
+			// RPC terminating itself so it should not block this call until the RPC interface disposed.
```

Comment:
This shouldn't say `disposed` I think. https://github.com/zkSNACKs/WalletWasabi/pull/4604#discussion_r512018193

Reply:
I changed it to stopping according to this https://github.com/molnard/WalletWasabi/blob/8c60a9aab5a650699df939bf565d327cb602bde5/WalletWasabi.Gui/Global.cs#L648